accounts/keystore/keystore.go

```diff
@@ -137,8 +138,10 @@ func (ks *KeyStore) refreshWallets() {
 	accs := ks.cache.accounts()
 
 	// Transform the current list of wallets into the new one
-	wallets := make([]accounts.Wallet, 0, len(accs))
-	events := []accounts.WalletEvent{}
+	var (
+		wallets = make([]accounts.Wallet, 0, len(accs))
+		events  []accounts.WalletEvent
+	)
```

Comment:
Why do we need this change?

Reply:
This came from https://github.com/ethereum/go-ethereum/pull/19079. If this is not necessary i'll remove it!

Reply:
It's fine.